tests/test_dtcwt.py

```diff
@@ -65,13 +66,71 @@ def test_inv(level1, qshift, J, T, alternate_gh, normalize):
     torch.testing.assert_close(Xt, X_rec)
 
 
-@pytest.mark.parametrize("include_scale", [False, [0, 0, 1]])
+@pytest.mark.parametrize("normalize", [False, True])
 @pytest.mark.parametrize("skip_hps", [False, [0, 1, 0]])
-def test_skip_hps(skip_hps, include_scale):
+def test_skip_hps(skip_hps, normalize):
```

Comment:
please add a docstring or comment explaining what you're testing. it seems like you are testing for shape and then testing for value